config/default/kustomization.yaml

```diff
@@ -25,18 +25,18 @@ resources:
 # [CERTMANAGER] To enable cert-manager, uncomment all sections with 'CERTMANAGER'. 'WEBHOOK' components are required.
 #- ../certmanager
 # [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
-#- ../prometheus
+- ../prometheus
 # [METRICS] To enable the controller manager metrics service, uncomment the following line.
-#- metrics_service.yaml
+- metrics_service.yaml
```

Comment:
this service points at port 8080. What is our prometheus port.

I am not sure you need this. it the pod wasn't individually reachable then service might be useful but directly scraping the pods is usually preferred. (when should I use a service is a fun quesiton for observability channel)

Reply:
ah okay the patch below sets --metrics-bind-address=:8080 so it was at least hitting the right port. Still not sure why we'd need a service.

Reply:
still think you can just always light up the metrics mind address rather than rely ing on this patch but can be subsqeuent pr. .